mods/seasonal/moves.js

```diff
@@ -55,6 +57,7 @@ exports.BattleMovedex = {
 		accuracy: 100,
 		basePower: 0,
 		category: "Physical",
+		shortDesc: "Does damage equal to the user's HP. User faints. Replacement is fully healed.",
```

Comment:
Actually this move does damage equal to the amount of damage the pokemon has taken.

`damage = pokemon.maxhp - pokemon.hp;`